### ISSUE TITLE
can you provided a example

### ISSUE THREAD

**Issue by xiguaaxigua** [jaketrent/react-dnd-text-dragpreview#1]
Hi man, wonderful project! 
but can you provided a simple example about how to use it??
thanks!

**Comment by BaooLi** [comment]
yes,I have the same prombles as you,I dont't konw how to use it,please show me the demo。
By the way,
```
import { ItemTypes } from './constants'
```
where are the './constants' come from?
